test/functional/index.js

```diff
@@ -3,29 +3,26 @@ define(function (require) {
 
   const bdd = require('intern!bdd');
   const intern = require('intern');
-  const initCallbacks = [];
 
-  function onInit(callback) {
-    initCallbacks.push(callback);
-  }
-
-  global.__kibana__intern__ = { intern, bdd, onInit };
+  global.__kibana__intern__ = { intern, bdd };
 
   bdd.describe('kibana', function () {
     bdd.before(function () {
-      initCallbacks.forEach(callback => {
-        callback.call(this);
-      });
+      this.PageObjects.init(this.remote);
     });
 
     require([
-      'intern/dojo/node!../support/index',
+      'intern/dojo/node!../support/page_objects.js',
+      'intern/dojo/node!../support',
       'intern/dojo/node!./apps/discover',
-      'intern/dojo/node!./status_page',
-      'intern/dojo/node!./apps/management',
-      'intern/dojo/node!./apps/visualize',
-      'intern/dojo/node!./apps/console',
-      'intern/dojo/node!./apps/dashboard'
-    ], function () {});
+      // TODO: Convert the rest of these to use PageObjects.
+      // 'intern/dojo/node!./status_page',
+      // 'intern/dojo/node!./apps/management',
+      // 'intern/dojo/node!./apps/visualize',
+      // 'intern/dojo/node!./apps/console',
+      // 'intern/dojo/node!./apps/dashboard'
+    ], PageObjects => {
+      this.PageObjects = PageObjects;
```

Comment:
I think using a `let PageObjects` in an higher scope would be more clear than using `this`